kitchen/project.clj

```diff
@@ -14,8 +14,8 @@
 ;; limitations under the License.
 ;;
 (defproject kitchen "0.1.0-SNAPSHOT"
-  :dependencies [^{:voom {:repo "https://github.com/twosigma/jet.git" :branch "waiter-patch"}}
```

Comment:
Jet PR merged, can you change this back to waiter-patch?

Reply:
Done.